backend/app/api/endpoints/adapter/models.py

```
@@ -537,6 +537,163 @@ def _test_embedding_connection(
         }
 
 
+@router.post("/fetch-available-models")
+def fetch_available_models(
+    fetch_data: dict,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.10)

Unused function argument: `current_user`

(ARG001)